data_process/convert_robotwin2lerobot_pi.py

```
@@ -282,6 +282,7 @@ def main(input_path, output_path):
             
         except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)